### DISCUSSION TITLE
Triga Films

### DISCUSSION THREAD

**Discussion by JPH71** [CodyBerenson/PGMA-Modernized#83] [General]
Guys keep a look out for a site that list Triga Films - a lot of their recent stuff and quite a bit of their old titles don't match...
and it's rubbing my tits raw.....

**Comment by joshallen13** [comment]
I'm pulling out what little hair I have left over getting these to match, especially since I can find their catalog spread out over several websites. Not one seems to be a comprehensive listing though, and the ones that are fairly close, don't return the results when searching those sites directly. (I'm looking at you Homoactive and GayRado LOL).

[Simply Adult - Triga Films](https://www.simply-adult.com/Gay-Dvd-Geezer-Cabs-Triga-Films.html)

This site seems to have a healthy number of videos for Triga, however they don't have the years listed as far as I can see, and I've been having a little difficulty with my GEVI not wanting to match titles with the generic 1900 release year, so I don't know if that will be a hindrance to using this as a potential site for scraping. 

... and no one likes a raw tit.. that hurts like a mother! ;)

**Comment by JPH71** [comment]
The 1900 should work if there is no date on the website...

**Reply by joshallen13** [reply]
I was getting a number of videos that had dates on the website, but being too lazy to look them up had marked them with the 1900 as a placeholder. Recently I had to reindex my library and a number of these that were marked as 1900 but had actual years in GEVI would not match. Not sure if that is just me, or if anyone else experienced this.

**Comment by JPH71** [comment]
That is correct, if GEVI has a date it will not match..
